etc/config/fortran.amazon.properties

```diff
@@ -0,0 +1,114 @@
+compiler.gfortran520.name=x86-64 gfortran 5.2
+compiler.gfortran530.exe=/opt/compiler-explorer/gcc-5.3.0/bin/gfortran
+compiler.gfortran530.name=x86-64 gfortran 5.3
+compiler.gfortran540.exe=/opt/compiler-explorer/gcc-5.4.0/bin/gfortran
+compiler.gfortran540.name=x86-64 gfortran 5.4
+compiler.gfortran550.exe=/opt/compiler-explorer/gcc-5.5.0/bin/gfortran
+compiler.gfortran550.name=x86-64 gfortran 5.5
+compiler.gfortran6.exe=/opt/compiler-explorer/gcc-6.1.0/bin/gfortran
+compiler.gfortran6.name=x86-64 gfortran 6.1
+compiler.gfortran62.exe=/opt/compiler-explorer/gcc-6.2.0/bin/gfortran
+compiler.gfortran62.name=x86-64 gfortran 6.2
+compiler.gfortran63.exe=/opt/compiler-explorer/gcc-6.3.0/bin/gfortran
+compiler.gfortran63.name=x86-64 gfortran 6.3
+compiler.gfortran71.exe=/opt/compiler-explorer/gcc-7.1.0/bin/gfortran
+compiler.gfortran71.name=x86-64 gfortran 7.1
+compiler.gfortran72.exe=/opt/compiler-explorer/gcc-7.2.0/bin/gfortran
+compiler.gfortran72.name=x86-64 gfortran 7.2
+compiler.gfortran73.exe=/opt/compiler-explorer/gcc-7.3.0/bin/gfortran
+compiler.gfortran73.name=x86-64 gfortran 7.3
+compiler.gfortran81.exe=/opt/compiler-explorer/gcc-8.1.0/bin/gfortran
+compiler.gfortran81.name=x86-64 gfortran 8.1
+compiler.gfortransnapshot.exe=/opt/compiler-explorer/gcc-snapshot/bin/gfortran
+compiler.gfortransnapshot.name=x86-64 gfortran (trunk)
+compiler.gfortransnapshot.alias=g7snapshot
+
+###############################
+# GCC for ppc
+group.gfortran_ppc.compilers=ppcgfortran48:ppc64legfortran630
+group.gfortran_ppc.needsMulti=false
+group.gfortran_ppc.supportsBinary=false
+group.gfortran_ppc.groupName=PowerPC GCC
+compiler.ppcgfortran48.exe=/opt/compiler-explorer/powerpc/gcc-4.8.5/bin/powerpc-unknown-linux-gnu-gfortran
+compiler.ppcgfortran48.name=PowerPC gfortran 4.8.5
+compiler.ppc64legfortran630.exe=/opt/compiler-explorer/powerpc64le/gcc-6.3.0/powerpc64le-unknown-linux-gnu/bin/powerpc64le-unknown-linux-gnu-gfortran
+compiler.ppc64legfortran630.name=PowerPC 64 6.3.0
+
+###############################
+# GCC for ARM
+group.gfortran_arm.compilers=armgfortran454:armgfortran464:aarchgfortran54:armhfgfortran54:armf541:armgfortran630:armf710:arm64gfortran630
+group.gfortran_arm.groupName=ARM GCC
+compiler.aarchgfortran54.exe=/opt/compiler-explorer/arm64/gcc-5.4.0/aarch64-unknown-linux-gnueabi/bin/aarch64-unknown-linux-gnueabi-gfortran
+compiler.aarchgfortran54.name=ARM64 gfortran 5.4 (linux)
+compiler.aarchgfortran54.alias=aarchg48
+compiler.armhfgfortran54.exe=/opt/compiler-explorer/arm/gcc-5.4.0/arm-unknown-linux-gnueabihf/bin/arm-unknown-linux-gnueabihf-gfortran
+compiler.armhfgfortran54.name=ARM gfortran 5.4 (linux)
+compiler.armhfgfortran54.alias=armhfg482
+compiler.armf541.exe=/opt/compiler-explorer/gcc-arm-none-eabi-5_4-2016q3/bin/arm-none-eabi-gfortran
+compiler.armf541.name=ARM gfortran 5.4.1 (none)
+compiler.armgfortran454.exe=/opt/compiler-explorer/arm/gcc-4.5.4/bin/arm-unknown-linux-gnueabi-gfortran
+compiler.armgfortran454.alias=armg453
```

Comment:
likewise these aliases; armg453 is actually the arm C++ compiler so this might cause problems.